### PR TITLE
blst: init at 0.3.10

### PR DESCRIPTION
**Pull request by iquerejeta**
###### Description of changes

New package support of `blst`. `blst` (pronounced 'blast') is a BLS12-381 signature library focused on performance and security. It is written in C and assembly.

homepage: https://github.com/supranational/blst

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/blst/default.nix
Comment (drupol) [inline pkgs/development/libraries/blst/default.nix:3]: I think we should use the `finalAttrs` pattern here.

Since this is your first PR, let me share a couple of useful links regarding this:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

File: pkgs/development/libraries/blst/default.nix
Comment (drupol) [inline pkgs/development/libraries/blst/default.nix:10]: Use `hash` instead of `sha256`

File: pkgs/development/libraries/blst/default.nix
Comment (drupol) [inline pkgs/development/libraries/blst/default.nix:13]: Missing hooks.

File: pkgs/development/libraries/blst/default.nix
Comment (drupol) [inline pkgs/development/libraries/blst/default.nix:14]: Missing hooks

File: pkgs/development/libraries/blst/default.nix
Comment (drupol) [inline pkgs/development/libraries/blst/default.nix:23]: Consider adding yourself as a maintainer.

**Review by drupol — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/blst/default.nix
Comment (drupol) [inline pkgs/development/libraries/blst/default.nix:17]:
```suggestion
  installPhase = ''
    runHook preInstall

    mkdir -p $out/lib
    cp ./libblst.a $out/lib/

    runHook postInstall
  '';
```

**Review by drupol — CHANGES_REQUESTED** [review]
Some minor things needs to be changed, I added the changes myself so you know for the next time.

Please include those changes to proceed further.

**Review by drupol — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/blst/default.nix
Comment (iquerejeta) [reply pkgs/development/libraries/blst/default.nix:10]: I'm getting a `error: cannot find flake 'flake:blst' in the flake registries`. Do you know why that might be?

Comment (yvan-sraka) [reply pkgs/development/libraries/blst/default.nix:10]: Here is the set of commands I used to test that this PR still works:
```
yvan@X230 ~ % gh repo clone iquerejeta/nixpkgs
yvan@X230 ~ % export NIX_PATH=nixpkgs=$(pwd)/nixpkgs
yvan@X230 ~ % nix repl                   
Welcome to Nix 2.13.3. Type :? for help.

nix-repl> pkgs = import <nixpkgs> {}
nix-repl> pkgs.libblst               
error: hash '01m28xxmm1x7riwyax7v9rycwl5isi06h2b2hl4gxnnylkayisn5' does not include a type, nor is the type otherwise known from context
«derivation 
```
You should have written ` hash = "sha256:01m28xxmm1x7riwyax7v9rycwl5isi06h2b2hl4gxnnylkayisn5";` (with `sha256` prefix) 😉

File: pkgs/development/libraries/blst/default.nix
Comment (yvan-sraka) [inline pkgs/development/libraries/blst/default.nix:10]:
```suggestion
    hash = "sha256:01m28xxmm1x7riwyax7v9rycwl5isi06h2b2hl4gxnnylkayisn5";
```

**Comment by yvan-sraka** [comment]
> Some minor things needs to be changed, I added the changes myself so you know for the next time.
> 
> Please include those changes to proceed further.

@drupol the review suggestions/comments have been addressed, thanks for your help through merging this :)

File: pkgs/development/libraries/blst/default.nix
Comment (yvan-sraka) [reply pkgs/development/libraries/blst/default.nix:10]: `nix build blst` isn't the right way of triggering the evaluation of the derivation … but the technic that mentioned @drupol to obtain the hash: TOFU ([Trust on first use](https://en.wikipedia.org/wiki/Trust_on_first_use)) as _“Trust the hash of a source for the first time when you fetch it, and fix it in the nix expression”_ is the right way of obtaining an SRI hash without bothering about understanding its format! Just use an empty string and iterate from the error! 😄

Comment (yvan-sraka) [reply pkgs/development/libraries/blst/default.nix]: Really? I made the change I suggested and then everything works as expected:

```
yvan@X230 ~ % nix repl
Welcome to Nix 2.13.3. Type :? for help.

nix-repl> pkgs = import <nixpkgs> {} 

nix-repl> pkgs.libblst      
«derivation /nix/store/gb2i3jf25ir5d4v0z47s1jlpz9al32yw-blst-0.3.10.drv»

yvan@X230 ~ % nix-build /nix/store/gb2i3jf25ir5d4v0z47s1jlpz9al32yw-blst-0.3.10.drv
this derivation will be built:
  /nix/store/gb2i3jf25ir5d4v0z47s1jlpz9al32yw-blst-0.3.10.drv
these 2 paths will be fetched (0.44 MiB download, 3.33 MiB unpacked):
  /nix/store/c3f4jdwzn8fm9lp72m91ffw524bakp6v-stdenv-linux
  /nix/store/ghxnazmksdqhwynzbkqikaa6bikla655-source
copying path '/nix/store/ghxnazmksdqhwynzbkqikaa6bikla655-source' from 'https://cache.zw3rk.com'...
copying path '/nix/store/c3f4jdwzn8fm9lp72m91ffw524bakp6v-stdenv-linux' from 'https://cache.nixos.org'...
building '/nix/store/gb2i3jf25ir5d4v0z47s1jlpz9al32yw-blst-0.3.10.drv' on 'ssh://builder@x86_64-linux-2.zw3rk.com'...
copying path '/nix/store/wyzaa007anaxxhmrcbffm597v14n2mxs-linux-headers-6.1' from 'https://cache.nixos.org'...
copying path '/nix/store/6g6d4la7xsizvr8qg91f56jiqx149iqq-binutils-2.40-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/45a55rzx3k794626g8adslzc6557gh0j-expand-response-params' from 'https://cache.nixos.org'...
copying path '/nix/store/fhzz4yrdy17czwc9i4swhlpcp445inzb-binutils-2.40' from 'https://cache.nixos.org'...
copying path '/nix/store/pqnd39aq2sksad2zvswjcpkqdc7ig3f9-glibc-2.35-224-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/wx4nsqbssywgx9zkabwgl9sxm5l61ni3-gcc-12.2.0' from 'https://cache.nixos.org'...
copying path '/nix/store/i2pdyabq6nrrnisbkma71h42fw6ha0y6-binutils-wrapper-2.40' from 'https://cache.nixos.org'...
copying path '/nix/store/bfbp3ypd9nm3fapz634gvvs738blrl0y-gcc-wrapper-12.2.0' from 'https://cache.nixos.org'...
copying path '/nix/store/c3f4jdwzn8fm9lp72m91ffw524bakp6v-stdenv-linux' from 'https://cache.nixos.org'...
copying 0 paths...
unpacking sources
unpacking source archive /nix/store/ghxnazmksdqhwynzbkqikaa6bikla655-source
source root is source
patching sources
configuring
no configure script, doing nothing
building
+ gcc -O -fno-builtin -fPIC -Wall -Wextra -Werror -D__ADX__ -mno-avx -c ./src/server.c
+ gcc -O -fno-builtin -fPIC -Wall -Wextra -Werror -D__ADX__ -mno-avx -c ./build/assembly.S
+ ar rc libblst.a assembly.o server.o
running tests
no Makefile or custom checkPhase, doing nothing
installing
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/maryx6yj7lk392i1z1cxc9g3sqb8h75a-blst-0.3.10
checking for references to /build/ in /nix/store/maryx6yj7lk392i1z1cxc9g3sqb8h75a-blst-0.3.10...
patching script interpreter paths in /nix/store/maryx6yj7lk392i1z1cxc9g3sqb8h75a-blst-0.3.10
stripping (with command strip and flags -S) in  /nix/store/maryx6yj7lk392i1z1cxc9g3sqb8h75a-blst-0.3.10/lib
copying 1 paths...
copying path '/nix/store/maryx6yj7lk392i1z1cxc9g3sqb8h75a-blst-0.3.10' from 'ssh://builder@x86_64-linux-2.zw3rk.com'...
/nix/store/maryx6yj7lk392i1z1cxc9g3sqb8h75a-blst-0.3.10
```

Comment (yvan-sraka) [reply pkgs/development/libraries/blst/default.nix]: It appears both work, but yes the empty hash suggest to rather to use `-` separator:

```
% nix repl                                                             
Welcome to Nix 2.13.3. Type :? for help.

nix-repl> pkgs = import <nixpkgs> {}

nix-repl> pkgs.libblst               
warning: found empty hash, assuming 'sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA='
«derivation /nix/store/9p5xb8ygbgf2pf2mq1w1vhrcghflx8wd-blst-0.3.10.drv»

yvan@X230 ~/nixpkgs (git)-[master] % nix-build /nix/store/9p5xb8ygbgf2pf2mq1w1vhrcghflx8wd-blst-0.3.10.drv
these 2 derivations will be built:
  /nix/store/8vdc08f3b8cs5i1zkrpzflk7aw7fcrwa-source.drv
  /nix/store/9p5xb8ygbgf2pf2mq1w1vhrcghflx8wd-blst-0.3.10.drv
these 20 paths will be fetched (4.34 MiB download, 18.12 MiB unpacked):
  /nix/store/0k5dhghrssfd0fs0bkvaw698w6666v9g-mirrors-list
  /nix/store/2nmqzvgf4rnjvpjsdkycf41lf63br42a-unzip-6.0
  /nix/store/2zc9s4cbdq1cnzimhqj5mv71qa3xpcp7-libkrb5-1.20.1-dev
  /nix/store/7vwvn3d5yb5yb3vdv9iv4321ckx4paf9-nss-cacert-3.86
  /nix/store/8sqpilc7xz92rs2d3i2l54ahhbpir1jg-curl-7.88.0-bin
  /nix/store/aq06nz8mq5nxkccasnj94zdk9pfqi2vs-openssl-3.0.8
  /nix/store/dawkj271851d8kpxz97grfsryjzlhsjj-glibc-locales-2.35-224
  /nix/store/f0j5grk3yf8pnml676jrcph4a88g8c2g-curl-7.88.0-dev
  /nix/store/facpy8mrdga06s4vcfzampl1b3rdh2kx-libssh2-1.10.0
  /nix/store/hf066cp95jiq13xl46s2zrdi94qvqlvv-openssl-3.0.8-dev
  /nix/store/hixx5vvivfzwj8cil0a5sfxbqp7lvb1d-nghttp2-1.51.0
  /nix/store/hwka9k94awxf8yq99rp7x0zbwksha5a7-curl-7.88.0
  /nix/store/jij4waw18na5smksrxfwzky7nd0hf2m6-curl-7.88.0-man
  /nix/store/mm2g3f5s75618ck1q4niyl1k57vx88ha-nghttp2-1.51.0-bin
  /nix/store/mpds2vrlb6m0yxs23fi6gs4z7y7h7ljr-nghttp2-1.51.0-lib
  /nix/store/qgqsyg7gmqy3fz6shvzhcc7m7pkfw2ig-zlib-1.2.13-dev
  /nix/store/rwfwqrvs6sjflbzc8wnkj57mnv94s40s-libkrb5-1.20.1
  /nix/store/ylgmlavny0m9afvqmki0w7wciprhvk69-nghttp2-1.51.0-dev
  /nix/store/zfsf3gffd6myvlkly4hb4cdndbb7dyic-openssl-3.0.8-bin
  /nix/store/zs3v6nqczpgs8v9pnk528p6kkr8h46f2-libssh2-1.10.0-dev
copying path '/nix/store/dawkj271851d8kpxz97grfsryjzlhsjj-glibc-locales-2.35-224' from 'https://cache.nixos.org'...
copying path '/nix/store/0k5dhghrssfd0fs0bkvaw698w6666v9g-mirrors-list' from 'https://cache.nixos.org'...
copying path '/nix/store/7vwvn3d5yb5yb3vdv9iv4321ckx4paf9-nss-cacert-3.86' from 'https://cache.nixos.org'...
copying path '/nix/store/jij4waw18na5smksrxfwzky7nd0hf2m6-curl-7.88.0-man' from 'https://cache.nixos.org'...
copying path '/nix/store/rwfwqrvs6sjflbzc8wnkj57mnv94s40s-libkrb5-1.20.1' from 'https://cache.nixos.org'...
copying path '/nix/store/hixx5vvivfzwj8cil0a5sfxbqp7lvb1d-nghttp2-1.51.0' from 'https://cache.nixos.org'...
copying path '/nix/store/mm2g3f5s75618ck1q4niyl1k57vx88ha-nghttp2-1.51.0-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/mpds2vrlb6m0yxs23fi6gs4z7y7h7ljr-nghttp2-1.51.0-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/aq06nz8mq5nxkccasnj94zdk9pfqi2vs-openssl-3.0.8' from 'https://cache.nixos.org'...
copying path '/nix/store/2nmqzvgf4rnjvpjsdkycf41lf63br42a-unzip-6.0' from 'https://cache.nixos.org'...
copying path '/nix/store/qgqsyg7gmqy3fz6shvzhcc7m7pkfw2ig-zlib-1.2.13-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/2zc9s4cbdq1cnzimhqj5mv71qa3xpcp7-libkrb5-1.20.1-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/ylgmlavny0m9afvqmki0w7wciprhvk69-nghttp2-1.51.0-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/zfsf3gffd6myvlkly4hb4cdndbb7dyic-openssl-3.0.8-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/facpy8mrdga06s4vcfzampl1b3rdh2kx-libssh2-1.10.0' from 'https://cache.nixos.org'...
copying path '/nix/store/zs3v6nqczpgs8v9pnk528p6kkr8h46f2-libssh2-1.10.0-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/hwka9k94awxf8yq99rp7x0zbwksha5a7-curl-7.88.0' from 'https://cache.nixos.org'...
copying path '/nix/store/hf066cp95jiq13xl46s2zrdi94qvqlvv-openssl-3.0.8-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/8sqpilc7xz92rs2d3i2l54ahhbpir1jg-curl-7.88.0-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/f0j5grk3yf8pnml676jrcph4a88g8c2g-curl-7.88.0-dev' from 'https://cache.nixos.org'...
building '/nix/store/8vdc08f3b8cs5i1zkrpzflk7aw7fcrwa-source.drv'...

trying https://github.com/supranational/blst/archive/03b5124029979755c752eec45f3c29674b558446.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  526k    0  526k    0     0   779k      0 --:--:-- --:--:-- --:--:--  779k
unpacking source archive /build/03b5124029979755c752eec45f3c29674b558446.tar.gz
error: hash mismatch in fixed-output derivation '/nix/store/8vdc08f3b8cs5i1zkrpzflk7aw7fcrwa-source.drv':
         specified: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
            got:    sha256-xero1aTe2v4IhWIJaEDUsVDOfE77dOV5zKeHWntHogY=
error: 1 dependencies of derivation '/nix/store/9p5xb8ygbgf2pf2mq1w1vhrcghflx8wd-blst-0.3.10.drv' failed to build
```

Comment (yvan-sraka) [reply pkgs/development/libraries/blst/default.nix]: Also, hash is different 🤔 damn, I do not understand SRI …

File: pkgs/development/libraries/blst/default.nix
Comment (yvan-sraka) [inline pkgs/development/libraries/blst/default.nix:10]:
```suggestion
    hash = "sha256-xero1aTe2v4IhWIJaEDUsVDOfE77dOV5zKeHWntHogY=";
```

File: pkgs/development/libraries/blst/default.nix
Comment (yvan-sraka) [reply pkgs/development/libraries/blst/default.nix]: AFAIU `-` is the syntax of non SRI-encoded hash to nix https://github.com/NixOS/nix/blob/master/src/libutil/hash.cc#L123 and `:` means SRI-encoded so `base64`. Am I right?

Comment (drupol) [reply pkgs/development/libraries/blst/default.nix]: Ok I'm back.

First of all, sorry @iquerejeta , I gave you an wrong information by telling you to run `nix build blst`, it actually should have been `nix build .#libblst`.

Second, regarding the hash, I don't mind using `:` or `-`, but I do care about consistency. In `nixpkgs`, I've never seen anything else than `-`, so I would have the tendency to follow that pattern.

And lastly, thanks for implementing all the changes, everything seems to be ok now :)

**Review by drupol — CHANGES_REQUESTED** [review]
Thanks for implementing all the changes.

@iquerejeta you still need to add yourself as maintainers in the maintainer list.

See a similar example in here: https://github.com/NixOS/nixpkgs/pull/240312

Once it's done, don't forget to squash all your commits into one single commit.

After that I'll merge this PR immediately.

**Comment by yvan-sraka** [comment]
> Thanks for implementing all the changes.
> 
> @iquerejeta you still need to add yourself as maintainers in the maintainer list.
> 
> See a similar example in here: https://github.com/NixOS/nixpkgs/pull/240312
> 
> Once it's done, don't forget to squash all your commits into one single commit.
> 
> After that I'll merge this PR immediately.

I think that would be a good idea to also add myself to the maintainer list :)

File: maintainers/maintainer-list.nix
Comment (drupol) [inline maintainers/maintainer-list.nix:6517]:
```suggestion
    github = "iquerejeta";
```

File: maintainers/maintainer-list.nix
Comment (drupol) [inline maintainers/maintainer-list.nix:6516]: We also try to have attributes sorted alphabetically, but it's just a detail. If you modify this once again, please sort them.

**Comment by drupol** [comment]
> I think that would be a good idea to also add myself to the maintainer list :)

Are you already in the maintainers list ?

**Comment by drupol** [comment]
I'll set this PR to `draft` until everything is done (add @yvan-sraka as maintainer and squash all your commits). Once it's done, feel free to remove it from `draft` and I'll review it once again.

Thanks!

File: pkgs/development/libraries/blst/default.nix
Comment (yvan-sraka) [inline pkgs/development/libraries/blst/default.nix:37]:
```suggestion
    maintainers = with maintainers; [ iquerejeta yvan-sraka ];
```

File: pkgs/development/libraries/blst/default.nix
Comment (drupol) [inline pkgs/development/libraries/blst/default.nix:4]:
```suggestion
  pname = "blst";
  version = "0.3.10";
```

**Review by drupol — CHANGES_REQUESTED** [review]
There are still two minor things to fix:

1. Don't create the attribute `name` yourself, it will be created automatically from the `pname` and `version` attributes.
2. Your commit message should be: `blst: init at 0.3.10`

File: pkgs/development/libraries/blst/default.nix
Comment (drupol) [inline pkgs/development/libraries/blst/default.nix:10]:
```suggestion
    rev = "v${finalAttrs.version}";
```

**Review by drupol — CHANGES_REQUESTED** [review]
Last minute change

**Review by drupol — APPROVED** [review]
LGTM now !

Congratulations for your first PR !!! And thanks for your patience and bearing with me.

I'll wait for the test to be done and I merge this.

**Comment by drupol** [comment]
@yvan-sraka Should this be named `libblst` or `blst` ?

**Comment by iquerejeta** [comment]
Thanks to you for helping out and guiding me through this PR!

**Comment by drupol** [comment]
@GrahamcOfBorg build libblst
@GrahamcOfBorg test libblst

File: pkgs/development/libraries/blst/default.nix
Comment (SuperSandro2000) [inline pkgs/development/libraries/blst/default.nix:30]: I don't think that is doing anything when providing a custom buildPhase

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
.

File: maintainers/maintainer-list.nix
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:6515]: Please put this hunk into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

**Comment by yvan-sraka** [comment]
> @yvan-sraka Should this be named `libblst` or `blst` ?

Let's keep `libblst`, is there some naming convention in nixpkgs I should be aware of?

**Comment by drupol** [comment]
> > @yvan-sraka Should this be named `libblst` or `blst` ?
> 
> Let's keep `libblst`, is there some naming convention in nixpkgs I should be aware of?

No, not at all.

If we use `libblst`, then the `pname` should be `libbst` as well.

I just wanted to clarify that.

File: pkgs/development/libraries/blst/default.nix
Comment (iquerejeta) [reply pkgs/development/libraries/blst/default.nix:30]: Should I just remove this line then?

File: pkgs/top-level/all-packages.nix
Comment (drupol) [inline pkgs/top-level/all-packages.nix:355]: This is unrelated changes :-1:

File: pkgs/development/libraries/blst/default.nix
Comment (drupol) [reply pkgs/development/libraries/blst/default.nix:30]: Yes, remove it.

**Review by drupol — CHANGES_REQUESTED** [review]
The modifications in `pkgs/servers/endlessh-go/default.nix` should be removed from this commit. Once it's done, I can finally merge this PR.

**Comment by iquerejeta** [comment]
Something must have gone off with the rebase. Sorry. Removing those now

**Review by drupol — APPROVED** [review]
Ok everything's good. Let's wait for the test and merge it !

File: maintainers/maintainer-list.nix
Comment (drupol) [inline maintainers/maintainer-list.nix:17995]:
```suggestion
  yvan-sraka = {
```

**Review by drupol — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix
Comment (drupol) [reply maintainers/maintainer-list.nix:17995]: This is the reason why `eval` is broken

**Comment by drupol** [comment]
@GrahamcOfBorg build blst
@GrahamcOfBorg test blst

**Review by yvan-sraka — APPROVED** [review]

**Comment by drupol** [comment]
Thanks for your first contribution !!! Keep them coming !!!

**Pull request merged**
